tests/Fixer/FunctionNotation/NoSpacesAfterFunctionNameFixerTest.php

```diff
@@ -167,6 +167,21 @@ function TisMy    ($p1)
             '<?php $a()(1);',
             '<?php $a () (1);',
         ];
+
+        yield [
```

Comment:
https://3v4l.org/CuJvh should be tested as well

not sure about other expressions like `throw ();`

Reply:
LGTM